build.gradle

```diff
@@ -15,44 +15,32 @@
  * limitations under the License.
  */
 
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.10.0'
-        classpath ("org.ajoberstar:grgit:2.0.0")
-    }
-}
-
 plugins {
-    id 'com.github.hierynomus.license' version '0.13.1'
-    id("org.nosphere.apache.rat") version "0.3.1"
-    id "com.jfrog.artifactory" version "4.9.5"
-    id "io.spring.dependency-management" version "1.0.8.RELEASE"
+    id 'com.github.hierynomus.license' version '0.15.0'
+    id 'org.nosphere.apache.rat' version "0.5.2"
+    id "com.jfrog.artifactory" version "4.11.0"
+    id "org.ajoberstar.grgit" version "4.0.1"
+    id "io.spring.dependency-management" version "1.0.9.RELEASE"
     id 'maven-publish'
-    id 'java'
+    id 'java-library'
     id 'idea'
 }
 
-apply plugin: "org.ajoberstar.grgit"
-
 ext {
     versions = [
-            springcontext        : '4.3.3.RELEASE',
-            springboot           : '1.4.1.RELEASE',
-            findbugs             : '3.0.1'
+            springcontext: '5.2.3.RELEASE',
+            springboot   : '2.2.4.RELEASE',
+            findbugs     : '3.0.2'
     ]
     changesetId = grgit.head().id
 }
 
 group 'org.apache.fineract.cn'
-version '0.1.0-BUILD-SNAPSHOT'
+version 'spring_boot_2-SNAPSHOT'
 
 tasks.withType(JavaCompile) {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_13
+    targetCompatibility = JavaVersion.VERSION_13
```

Comment:
Why you want to upgrade Java version?
Anyone currently using Fineract-CN is running it on Java8 (as Spring Boot 1.x doesn't work above Java 8). Why you want them to make two big changes at the same time. Let's only upgrade Spring & Spring Boot in this iteration and let's leave upgrading everything else to the future.

Reply:
basically java 8 is not supported by oracle, I pretty much believe whoever is using java8 is not by choice. I can downgrade to java 11 which is LTS but so far I see no point in sticking to java8. 
It was a very clear decision within our team as well that no java services below 11 would be developed.

Again fineract-cn is not being installed as java software, its a container service, until and unless container is running fine, its an easy task to replace.  We are packaging runtime, thats the whole point of containerization.

Reply:
OK, let's stick to Java 11 then. Please downgrade to Java 11.